bindings/python/crocoddyl/multibody/contacts/contact-1d.cpp

```diff
@@ -0,0 +1,122 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2020-2021, LAAS-CNRS, University of Edinburgh
+// Copyright note valid unless otherwise stated in individual files.
+// All rights reserved.
+///////////////////////////////////////////////////////////////////////////////
+
+#include "python/crocoddyl/multibody/multibody.hpp"
+#include "crocoddyl/multibody/contacts/contact-1d.hpp"
+#include "python/crocoddyl/utils/deprecate.hpp"
+
+namespace crocoddyl {
+namespace python {
+
+void exposeContact1D() {
+  bp::register_ptr_to_python<boost::shared_ptr<ContactModel1D> >();
+
+#pragma GCC diagnostic push  // TODO: Remove once the deprecated FrameXX has been removed in a future release
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+
+  bp::class_<ContactModel1D, bp::bases<ContactModelAbstract> >(
+      "ContactModel1D",
+      "Rigid 1D contact model.\n\n"
+      "It defines a rigid 1D contact models (point contact) based on acceleration-based holonomic constraints, in x,z "
+      "directions.\n"
+      "The calc and calcDiff functions compute the contact Jacobian and drift (holonomic constraint) or\n"
+      "the derivatives of the holonomic constraint, respectively.",
+      bp::init<boost::shared_ptr<StateMultibody>, pinocchio::FrameIndex, double, std::size_t,
+               bp::optional<Eigen::Vector2d> >(
+          bp::args("self", "state", "id", "xref", "nu", "gains"),
+          "Initialize the contact model.\n\n"
+          ":param state: state of the multibody system\n"
+          ":param id: reference frame id of the contact\n"
+          ":param xref: contact position used for the Baumgarte stabilization\n"
+          ":param nu: dimension of control vector\n"
+          ":param gains: gains of the contact model (default np.matrix([0.,0.]))"))
+      .def(bp::init<boost::shared_ptr<StateMultibody>, pinocchio::FrameIndex, double, bp::optional<Eigen::Vector2d> >(
+          bp::args("self", "state", "id", "xref", "gains"),
+          "Initialize the contact model.\n\n"
+          ":param state: state of the multibody system\n"
+          ":param id: reference frame id of the contact\n"
+          ":param xref: contact position used for the Baumgarte stabilization\n"
+          ":param gains: gains of the contact model (default np.matrix([0.,0.]))"))
+      .def(bp::init<boost::shared_ptr<StateMultibody>, FrameTranslation, std::size_t, bp::optional<Eigen::Vector2d> >(
+          bp::args("self", "state", "xref", "nu", "gains"),
```

Comment:
```suggestion
          bp::args("self", "state", "id", "nu", "gains"),
```

Please also update the documentation in line 49